assets/javascripts/discourse/components/ai-llms-list-editor.gjs

```diff
@@ -39,60 +90,118 @@ export default class AiLlmsListEditor extends Component {
       @path="/admin/plugins/{{this.adminPluginNavManager.currentPlugin.name}}/ai-llms"
       @label={{i18n "discourse_ai.llms.short_title"}}
     />
-    <section class="ai-llms-list-editor admin-detail pull-left">
-
+    <section class="ai-llm-list-editor admin-detail">
       {{#if @currentLlm}}
-        <AiLlmEditor @model={{@currentLlm}} @llms={{@llms}} />
+        <AiLlmEditor
+          @model={{@currentLlm}}
+          @llms={{@llms}}
+          @llmTemplate={{@llmTemplate}}
+        />
       {{else}}
-        <div class="ai-llms-list-editor__header">
-          <h3>{{i18n "discourse_ai.llms.short_title"}}</h3>
-          {{#unless @currentLlm.isNew}}
-            <LinkTo
-              @route="adminPlugins.show.discourse-ai-llms.new"
-              class="btn btn-small btn-primary ai-llms-list-editor__new"
-            >
-              {{icon "plus"}}
-              <span>{{I18n.t "discourse_ai.llms.new"}}</span>
-            </LinkTo>
-          {{/unless}}
-        </div>
-
-        {{#if this.hasLLMElements}}
-          <table class="content-list ai-persona-list-editor">
-            <thead>
-              <tr>
-                <th>{{i18n "discourse_ai.llms.display_name"}}</th>
-                <th>{{i18n "discourse_ai.llms.provider"}}</th>
-                <th>{{i18n "discourse_ai.llms.enabled_chat_bot"}}</th>
-                <th></th>
-              </tr>
-            </thead>
-            <tbody>
-              {{#each @llms as |llm|}}
-                <tr data-persona-id={{llm.id}} class="ai-llm-list__row">
-                  <td><strong>{{llm.display_name}}</strong></td>
-                  <td>{{i18n
-                      (concat "discourse_ai.llms.providers." llm.provider)
-                    }}</td>
-                  <td>
-                    <DToggleSwitch
-                      @state={{llm.enabled_chat_bot}}
-                      {{on "click" (fn this.toggleEnabledChatBot llm)}}
-                    />
-                  </td>
-                  <td>
-                    <LinkTo
-                      @route="adminPlugins.show.discourse-ai-llms.show"
-                      current-when="true"
-                      class="btn btn-text btn-small"
-                      @model={{llm}}
-                    >{{i18n "discourse_ai.llms.edit"}}</LinkTo>
-                  </td>
+        {{#if this.hasLlmElements}}
+          <section class="ai-llms-list-editor__configured">
+            <div class="admin-page-subheader">
```

Comment:
Please use the actual `AdminPageSubheader` component from core here, here is an example:

https://github.com/discourse/discourse/blob/c5a024f8dfcd0611f3661b59b0b7c8d5bc72fac8/app/assets/javascripts/admin/addon/components/admin-config-areas/flags.gjs#L72-L83

Also /t/129128 internally has docs